### PR TITLE
Dev stories: to tell the truth vs. to offer a comforting lie

### PR DESCRIPTION
**Pull request by paul-sheridan**
Update log:
- Renamed **to tell the truth vs. offering a comforting lie** to **to tell the truth vs. to offer a comforting lie**
- Improved motivations

### CODE REVIEW

File: notes/stories/television/tv-iclaudius.st.txt [hunk 1]
```diff
@@ -810,7 +810,7 @@ delivering bad news [Calpurnia was tasked with breaking the news to Claudius tha
 honorable suicide [Messalina was offered the dagger and urged to use it, but proved too weak of will to do so.]
 omens [An owl hooted five times and Herod dropped ill and died.]
 acting [Mnester was a renowned Greek actor.]
-to tell the truth vs. offering a comforting lie [Claudius' trusted advisers struggled with whether and how to break the news that Claudius' wife was serially unfaithful and, in the end, even treasonous to him.]
+tto tell the truth vs. to offer a comforting lie [Claudius' trusted advisers struggled with whether and how to break the news that Claudius' wife was serially unfaithful and, in the end, even treasonous to him.]
```
Comment (odinlake) [inline notes/stories/television/tv-iclaudius.st.txt:813]: Typo "tto" in theme name

File: notes/stories/television/tv-startrek-tng.st.txt [hunk 2]
```diff
@@ -9637,7 +9637,7 @@ transcendental being [The Q :: Q]
 
 :: Major Themes
 resisting a temptation [Amanda Rogers :: Amanda resists using Q powers]
-to tell the truth vs. offering a comforting lie [Picard :: Picard decides to tell Amanda the truth about the Q's intentions for her]
+tto tell the truth vs. to offer a comforting lie [Captain Picard decided to tell Amanda the truth about the Q's intentions for her.]
```
Comment (odinlake) [inline notes/stories/television/tv-startrek-tng.st.txt:9640]: stutter typo

File: notes/stories/television/tv-startrek-voyager.st.txt [hunk 3]
```diff
@@ -6246,7 +6246,7 @@ the nature of fear [Neelix and the Borg children discussed the good and bad aspe
 :: Major Themes
 facing a ship takeover [electromagnetic entity was taking over Voyager in Neelix's story]
 incorporeal being [electromagnetic space ghost]
-to tell the truth vs. offering a comforting lie [we were left to ponder whether Neelix had told the truth in the end after all]
+tto tell the truth vs. to offer a comforting lie [The viewer is left to ponder whether Neelix had told the truth in the end after all.]
```
Comment (odinlake) [inline notes/stories/television/tv-startrek-voyager.st.txt:6249]: Typo

**Review by odinlake — CHANGES_REQUESTED** [review]

File: notes/stories/television/tv-iclaudius.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-iclaudius.st.txt]: Whoopy. Fixed.

File: notes/stories/television/tv-startrek-tng.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-startrek-tng.st.txt]: Fixed

File: notes/stories/television/tv-startrek-voyager.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-startrek-voyager.st.txt]: And fixed

**Review by odinlake — APPROVED** [review]

**Pull request merged**